components/resources/ai_chat_ui_strings.grdp

```diff
@@ -37,7 +37,7 @@
     Summarize this page
   </message>
   <message name="IDS_CHAT_UI_ENABLE_QUESTIONS_DESC" desc="Description about suggested questions">
-    Brave AI can automatically suggest questions based on the content of the site. You can disable it later in Settings.
+    Leo can automatically suggest questions based on the content of the site. To do this, the page contents of your navigations will be sent to Leo to generate these suggestions while Leo is open. You can disable it later in Settings.
```

Comment:
`the page contents of your navigations` sounds odd to me. cc: @rmcfadden3

Reply:
@mkarolin — looks like this PR already merged, so I might be too late to make a suggestion. But yes, that copy is awkward. I'd suggest:

`To generate these suggestions, the contents of the page you're viewing will be sent to Leo (but only while the Leo chat window is open).`